lib/cell.rb

```diff
@@ -2,7 +2,7 @@
 
 class Cell
 
-  attr_reader :coordinate, :ship
+  attr_reader :coordinate, :ship, :hit
```

Comment:
Realized we could not check to validate b/c we forgot to make "hit" readable outside of the class. Oops, super buggy when I tried to call on it in the game. Hahaha